ISISComputingGroup/EPICS-IOC_Test_Framework
# Ticket 5412: Add Danfysik test that restarts IOC

See https://github.com/ISISComputingGroup/IBEX/issues/5412

To test:
* Run some of the following kinds of tests and confirm they still pass:
    * `ProcservLauncher` tests (`simple` or `lsicorr` use this)
    * `PythonIOCLauncher` tests (`refl`)
    * `IOCLauncher` tests (any other tests)
* Switch some `IOCLauncher` tests to be `ProcservLauncher` tests and confirm they still pass
* Run all new danfysik tests and confirm they pass
